analytics/analytics.go

```diff
@@ -107,13 +112,40 @@ func Initialize(n *core.IpfsNode, BTFSVersion string) {
 		dc.ArchType = runtime.GOARCH
 	}
 
-	go dc.collectionAgent()
+	go dc.collectionAgent(env)
 }
 
-func (dc *dataCollection) update() {
+func (dc *dataCollection) update(env cmds.Environment) {
 	var m runtime.MemStats
 	runtime.ReadMemStats(&m)
 
+	// get config info
+	var log = logging.Logger("cmd/btfs")
```

Comment:
Make this part of package and use `logging.Logger("analytics")` (ipfs convention)

Reply:
good catch, thanks

Reply:
good catch, thanks